src/ir/context.rs

```diff
@@ -1066,15 +1253,15 @@ impl BindgenContext {
     }
 
     /// Look up whether the item with `id` has vtable or not.
-    pub fn lookup_item_id_has_vtable(&self, id: &ItemId) -> bool {
+    pub fn lookup_item_id_has_vtable<Id: Into<ItemId>>(&self, id: Id) -> bool {
```

Comment:
I suppose `has_vtable` is defined only for types and calling it on a not type is bit smelly. Would it be hard to change this fn to only accept `TypeId`?

Reply:
Good call -- I'll give it a try.